stretch-assignment/index.js

```diff
@@ -0,0 +1,67 @@
+class Game{
+    constructor(){
+        this.big=document.querySelector('.blocks')
+        this.red=this.big.querySelector('.block--red')

```

Comment:
I was leaving a comment above about how the use of the `keydown` event and switch cases would be how you set up user based movement for a game, and see this. This is great work here! I love this.